### PR TITLE
onboarding: rename application

### LINKED ISSUE #259: App name on first pre onboarding screen
Change name of the app here to "Steps 4 Impact"
IMG_329F26B717A6-1

### PR DESCRIPTION
**Pull request by compnerd**
The application had to be renamed, update the name.  Resolves #259!

### CODE REVIEW

**Review by samisuteria — COMMENTED** [review]
Should use Application.name instead

**Comment by compnerd** [comment]
Yeah, we need to extend the infrastructure to do substitutions.

**Comment by samisuteria** [comment]
New localizable strings w/ formatting ftw

File: Steps4Impact/Supporting Files/hi.lproj/Localizable.strings [hunk 1]
```diff
@@ -43,7 +43,7 @@
 "Login.conditions" = "Terms and Conditions Apply";
 
 // Welcome
-"Welcome.title" = "Welcome to Steps for Change!";
+"Welcome.title" = "Welcome to %@";
```
Comment (compnerd) [inline Steps4Impact/Supporting Files/hi.lproj/Localizable.strings]: We should retain `!`

File: Steps4Impact/Supporting Files/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -43,7 +43,7 @@
 "Login.conditions" = "Terms and Conditions Apply";
 
 // Welcome
-"Welcome.title" = "Welcome to Steps for Change!";
+"Welcome.title" = "Welcome to %@";
```
Comment (compnerd) [inline Steps4Impact/Supporting Files/en.lproj/Localizable.strings]: We should retain `!`

Comment (samisuteria) [reply Steps4Impact/Supporting Files/en.lproj/Localizable.strings]: whoops - good thing for good review

**Review by compnerd — COMMENTED** [review]
LGTM

**Pull request merged**